pkg/server/grpc.go

```diff
@@ -74,7 +67,13 @@ func (s *gRPCServer) FindMalwareInfo(c context.Context, r *pb.MalwareRequest) (*
 			return
 		}
 
-		scanner := scan.New(s.options, s.yaraConfig, yaraScanner)
+		scanner := scan.New(s.options, s.yaraConfig, yaraScanner, r.ScanId)
+
+		res := jobs.StartStatusReporter(context.Background(), r.ScanId, scanner)
+		defer func() {
+			res <- err
+			close(res)
+		}()
```

Comment:
We actually need to do that as the very first thing (as it used to be)
`StartStatusReport` will report an error back to the console if we failed to create a Yara scanner, preventing unnecessary retries (if it fails at that stage, there is little chance retrying will solve anything)

Reply:
Good catch.... will make the change